app/Http/Controllers/WorkflowController.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\EvidencioAPI;
+use App\Workflow;
+use App\LoadedEvidencioModel;
+use App\Step;
+use App\Field;
+use App\Option;
+
+/**
+ * WorkflowController class, handles database- and API-calls for Workflowpage.
+ */
+class WorkflowController extends Controller
```

Comment:
The class shares a lot of similarity with the Designer Controllers. Maybe they should be merged together in the future, but I understand that currently there are more important things to do.

Reply:
True, I'll talk about it with the Patient side